src/parameters.rs

```diff
@@ -272,6 +273,7 @@ pub struct FinishDepositEthCallArgs {
 pub struct InitCallArgs {
     pub prover_account: AccountId,
     pub eth_custodian_address: AccountId,
+    pub metadata: FungibleTokenMetadata,
```

Comment:
Changing this struct means we also need to update the json library:

- https://github.com/aurora-is-near/aurora.js/blob/master/src/engine.ts#L153
- https://github.com/aurora-is-near/aurora.js/blob/master/src/schema.ts#L122
- https://github.com/aurora-is-near/aurora.js/blob/master/src/schema.ts#L257

Reply:
Adde issue: https://github.com/aurora-is-near/aurora.js/issues/7